pt/lessons/basics/pipe-operator.md

```diff
@@ -50,7 +50,7 @@ true
 
 ## Boas Práticas
 
-Se a aridade de uma função é mais do que 1, certifique-se de usar parêntesis. Isso não importa muito para Elixir, porém é importante para outros programadores que podem interpretar mal o seu código. Mas ainda assim, a ordem é importante com o operador pipe. Se tomarmos o nosso terceiro exemplo, e retirar os parêntesis do `String.ends_with?/2`, nos encontramos com o seguinte aviso:
+Se a aridade de uma função é mais do que 1, certifique-se de usar parênteses. Isso não importa muito para Elixir, porém é importante para outros programadores que podem interpretar mal o seu código. Mas ainda assim, a ordem é importante com o operador pipe. Se tomarmos o nosso terceiro exemplo, e retirarmos os parênteses do `String.ends_with?/2`, receberemos o seguinte aviso:
```

Comment:
I understand that the translation should match the english version, but `receber um aviso` makes more sense in this context for Portuguese speakers.